build.gradle

```diff
@@ -1,4 +1,4 @@
-ext.projectVersion = '1.4.0-SNAPSHOT'
+ext.projectVersion = '1.5.0-SNAPSHOT'
```

Comment:
If we created  `1.5-rc` branch, then master should be next minor snapshot, so `1.6.0-SNAPSHOT`, correct?